third_party/nvidia/lib/TritonNVIDIAGPUToLLVM/ReduceOpToLLVM.cpp

```diff
@@ -453,6 +453,12 @@ struct ReduceOpConversion
         for (size_t j = 0; j < resultElems; ++j) {
           SmallVector<Value> readIdx = resultIndices[j];
           readIdx.insert(readIdx.begin() + op.getAxis(), i32_val(0));
+          for (size_t i = 0, e = readIdx.size(); i < e; ++i) {
```

Comment:
Is it possible to use a different iterator name to avoid confusion with the outer loop ?

Reply:
Also, I would just do ` < readIdx.size()` instead of using `e`